Makefile

```diff
@@ -181,3 +181,25 @@ restore-postgres: deploy-init ## make qa restore-postgres DB_INSTANCE_GUID="<cf
 	$(eval export TF_VAR_paas_restore_db_from_point_in_time_before=$(BEFORE_TIME))
 	echo "Restoring apply-postgres-${APP_NAME_SUFFIX} from $(TF_VAR_paas_restore_db_from_db_instance) before $(TF_VAR_paas_restore_db_from_point_in_time_before)"
 	make ${APP_ENV} deploy
+
+enable-maintenance: ## make enable-maintenance CONFIRM_MAINTENANCE=y
+	$(if $(CONFIRM_MAINTENANCE), , $(error Maintenance mode can only be enabled with CONFIRM_MAINTENANCE))
+	# cf target -s bat-prod
+	cf target -s bat-qa
```

Comment:
Probably already on your list — could we parameterize `${APP_ENV}` here and below?

Reply:
We discussed it with @cwrw and thought of doing prod only for now. Parameterising would involve adding a level of abstraction for the domain as prod is www and not `$APP_ENV`. There may be little value in adding complexity as this will only be used for prod. Happy to discuss it though.

Reply:
Yea agreed to have only for prod for now, and iterate later as the config is different across both and would add complexity to support that

Reply:
The value of having something which can switch envs is that we can test it in non-prod environments. Agree that getting prod out is the required thing, tho! Is it just the `www`?

Reply:
Alright alright it's done :-)